_posts/2020-04-13-squeezing-the-most-out-of-your-pebble.md

```diff
@@ -0,0 +1,132 @@
+Monzo Debit card payments | [Timeline Bounce Server](https://github.com/Willow-Systems/timeline-bounce-server)
+New Hot item on hacker news | IFTTT: Hacker News to Timeline
+
+This keeps my timeline full of interesting stuff to read when I choosed to. Only BBC breaking news creates a notification, the rest is passive and I either read it when it pops up in [timeline peek](https://i.redd.it/7l87rqv4y9n41.png) or when I scroll back through timeline past (By mapping the 'up' quicklaunch to timeline past)   
+
+To get going with timeline you'll need [this app to generate a 'token'](https://apps.rebble.io/en_US/application/5d9ac26dc393f54d6b5f5445) and to follow [this guide](https://willow.systems/integrate-Pebble-with-ifttt-once-again/) or - if you prefer pictures - [this guide](https://imgur.com/a/IWG28JA). It helps if you're familiar with [If-This-Then-That](https://ifttt.com) too.
+
+# Developing New Apps
+Do you know how to program in C or Javascript? Maybe you want to learn? Developing on the Pebble is super fun and between the #app-dev channel on [the Discord](https://rebble.io/Discord) and [the documentation](https://developer.rebble.io/developer.Pebble.com/index.html) there's plenty of support still.   
+
+To get started you'll need the SDK. For which there are now a few options:   
+
+   - [Install to your computer natively](https://old.Reddit.com/r/Pebble/comments/9i9aqy/developing_for_Pebble_without_cloudPebble_windows/)   
+   - [Install to your computer with a script](https://gist.github.com/zeevro/ce20d0d74a869d73119e9d1522b7caa7)   
+   - [Use a docker image](https://hub.docker.com/r/dmorgan81/rebble/)   
+   - [Use a virtual machine](https://willow.systems/Pebble/vm/)   
+
+Once you've got an app or a watchface you'd like to publish, you can do so through the new [Rebble developer portal](https://rebble.io/submit). This developer portal is a work in progress, but the basics are there now. Watch this space!
+
+# Some other neat tips
+Here are a couple of neat Pebble tips that I've seen recently, that maybe you didn't already know:   
+
+   - When you start playing music on your phone, the stock music app will move to the top of your app list on your watch, making it easy to open by pressing select twice from your watchface   
+   - Holding back from a notification will toggle quiet time
+   - Enabling 'Stand-By Mode' from Settings -> System on your watch will disable bluetooth when your Pebble doesn't move for 30 minutes, saving battery power   
+
+# Darksky API Shutdown
+You might also have seen recently that Apple have purchased Darksky, one of the most popular weather API providers for Pebble apps and faces. Apple are going to shut down the API June 2021, and have already stopped issuing new API keys. Whilst this won't affect the Rebble weather service, watchfaces that require this API will no longer work properly. There's plenty of time to come up with a replacement solution though, so I'm hopeful.   
```

Comment:
can we clarify `no longer work properly`? Seems easy to interpret as "will no longer work properly now" vs. in 2021? :hourglass_flowing_sand: